src/tools/clippy/clippy_lints/src/unnested_or_patterns.rs

```diff
@@ -7,13 +7,13 @@ use clippy_utils::msrvs::{self, MsrvStack};
 use clippy_utils::over;
 use rustc_ast::PatKind::*;
 use rustc_ast::mut_visit::*;
-use rustc_ast::ptr::P;
 use rustc_ast::{self as ast, DUMMY_NODE_ID, Mutability, Pat, PatKind};
 use rustc_ast_pretty::pprust;
 use rustc_errors::Applicability;
 use rustc_lint::{EarlyContext, EarlyLintPass};
 use rustc_session::impl_lint_pass;
 use rustc_span::DUMMY_SP;
+use std::boxed::Box;
```

Comment:
Why is this necessary?

Reply:
to shadow `use rustc_ast::PatKind::*;` above